slack-plugin/src/main/kotlin/slack/gradle/SlackRootPlugin.kt

```diff
@@ -155,23 +155,30 @@ internal class SlackRootPlugin : Plugin<Project> {
     // Usage: ./gradlew clean buildHealth
     project.pluginManager.withPlugin("com.autonomousapps.dependency-analysis") {
       project.configure<DependencyAnalysisExtension> {
-        issues {
-          all {
-            onAny {
-              severity("warn")
-              exclude(
-                // This is a resources-only project.
-                ":libraries:l10n-strings"
-              )
-            }
-          }
-        }
+        issues { all { onAny { ignoreKtx(true) } } }
```

Comment:
btw, I _think_ you could use dot notation here? Or if not, it would be easy to support if you'd prefer that syntax for brevity.